apps/epl_ets/src/epl_ets_EPL.erl

```diff
@@ -25,8 +27,14 @@ init({tcp, http}, _Req, _Opts) ->
     {upgrade, protocol, cowboy_websocket}.
 
 websocket_init(_TransportName, Req, _Opts) ->
-    {ok, Req, undefined_state}.
-
+    {ok, Req, #state{}}.
+
+websocket_handle({text, NodeBin}, Req, 
+                 State = #state{ets_call_traced_nodes = TNodes}) ->
+    Node = erlang:binary_to_atom(NodeBin, latin1),
```

Comment:
That's a security hole - if someone DOS your service with malformed node names and you call `erlang:binary_to_atom/2` enough number of times you will kill the VM. I guess you MUST know the node name before you get this message, am I right? If so, then the atom representing the node in the system already exists and you should use `erlang:binary_to_existing_atom/2`.